charts/scalardl-auditor-monitoring/values.yaml

```diff
@@ -0,0 +1,693 @@
+          regex: (.+);
+          target_label: __tmp_hash
+          replacement: $1
+          action: replace
+        - source_labels: [__tmp_hash]
+          separator: ;
+          modulus: 1
+          target_label: __tmp_hash
+          replacement: $1
+          action: hashmod
+        - source_labels: [__tmp_hash]
+          separator: ;
+          regex: "0"
+          replacement: $1
+          action: keep
+        kubernetes_sd_configs:
+        - role: endpoints
+          kubeconfig_file: ""
+          follow_redirects: true
+          enable_http2: true
+          namespaces:
+            own_namespace: true
+
+  alertmanager:
+    enabled: false
+  kube-state-metrics:
+    enabled: true
+    customLabels:
+      app.kubernetes.io/app: "scalardl-auditor-monitoring"
+  prometheus-node-exporter:
+    enabled: false
+  prometheus-pushgateway:
+    enabled: false
+
+grafana:
+  enabled: true
+
+  extraLabels:
+    app.kubernetes.io/app: "scalardl-auditor-monitoring"
+
+  service:
+    type: ClusterIP
+    port: 3000
+
+  sidecar:
+    datasources:
+      enabled: true
+      label: grafana_datasource
+      labelValue: "1"
+    dashboards:
+      enabled: true
+      label: grafana_dashboard
+      labelValue: "1"
+
+  datasources:
+    datasources.yaml:
+      apiVersion: 1
+      datasources:
+        - name: Prometheus
+          type: prometheus
+          uid: prometheus
+          url: http://{{ .Release.Name }}-prometheus-server:9090/
+          access: proxy
+          editable: false
+          isDefault: false
+          jsonData:
+            timeInterval: 30s
+        - name: Loki
+          type: loki
+          uid: loki
+          url: http://{{ .Release.Name }}-loki:3100/
+          access: proxy
+          editable: false
+          isDefault: false
+
+  testFramework:
+    securityContext:
+      seccompProfile:
+        type: RuntimeDefault
+      runAsNonRoot: true
+    containerSecurityContext:
+      capabilities:
+        drop:
+          - ALL
+      allowPrivilegeEscalation: false
+
+loki:
+  enabled: true
+
+  loki:
+    podLabels:
+      app.kubernetes.io/app: "scalardl-auditor-monitoring"
+    auth_enabled: false
+    extraMemberlistConfig:
+      bind_addr:
+        - ${MY_POD_IP}
+    commonConfig:
+      replication_factor: 1
+    schemaConfig:
+      configs:
+        - from: "2024-04-01"
+          store: tsdb
+          object_store: s3
+          schema: v13
+          index:
+            prefix: loki_index_
+            period: 24h
+    pattern_ingester:
+      enabled: true
+    limits_config:
+      retention_period: 720h
+      discover_service_name: []
+    compactor:
+      retention_enabled: true
+      delete_request_store: s3
+
+  deploymentMode: SingleBinary
+
+  singleBinary:
+    replicas: 1
+    extraArgs:
+      - -config.expand-env=true
+    extraEnv:
+      - name: MY_POD_IP
+        valueFrom:
+          fieldRef:
+            fieldPath: status.podIP
+
+  minio:
```

Comment:
We use MinIO as a backend storage of Loki since we can deploy MinIO as a pod in the same Kubernetes cluster. This chart is an entry plan for monitoring. So, I choose this simple way.